aiida/storage/sqlite_zip/orm.py

```diff
@@ -325,13 +334,6 @@ def _cast_json_type(comparator: JSON.Comparator, value: Any) -> Tuple[ColumnElem
 
     @staticmethod
     def get_filter_expr_from_column(operator: str, value: Any, column) -> BinaryExpression:
-        """A method that returns an valid SQLAlchemy expression.
-
-        :param operator: The operator provided by the user ('==',  '>', ...)
-        :param value: The value to compare with, e.g. (5.0, 'foo', ['a','b'])
-        :param column: an instance of sqlalchemy.orm.attributes.InstrumentedAttribute or
-
-        """
```

Comment:
Any reason to remove the docstring?

Reply:
It was just a copy of the parent docstring, so then I think it is just general practice to have it "inherit" that?